lib/icasework/case.rb

```diff
@@ -60,6 +60,10 @@ def [](key)
       @hash[key]
     end
 
+    def to_hash
+      @hash
```

Comment:
AIUI this will always return all attributes because we call `load_additional_data!` on initialisation?